packages/module/patternfly-docs/content/examples/FilterSidePanel.md

```diff
@@ -106,7 +106,7 @@ class MockFilterSidePanelExample extends React.Component {
   
       return (
         <span>
-          <span className="pf-v5-u-screen-reader">{`${count} stars`}</span>
+          <span className="pf-v6-u-screen-reader">{`${count} stars`}</span>
```

Comment:
You can use `.pf-v6-screen-reader` which ships with the PF base styles. `.pf-v6-u-screen-reader` is a utility class so you have to import the accessibility utility class stylesheet to use it. There is nothing wrong with using the utility class, though if the only reason it's imported is to use this screen reader class, it may be an import you can remove.